pkg/visor/visor.go

```diff
@@ -160,195 +160,195 @@ func NewNode(cfg *Config, logger *logging.MasterLogger, restartCtx *restart.Cont
 	tmConfig := &transport.ManagerConfig{
 		PubKey:          pk,
 		SecKey:          sk,
-		DefaultNodes:    cfg.TrustedNodes,
+		DefaultVisors:   cfg.TrustedNodes,
```

Comment:
I think we need to rename `TrustedNodes` to `TrustedVisors`.

Reply:
After renaming `TrustedNodes` to `TrustedVisors` we need to make sure that config files in both repositories are renamed as well.

Reply:
Updated config files to `TrustedVisors`